1.9/administration/installing/custom/system-requirements/index.md

```diff
@@ -222,6 +222,9 @@ On each of your cluster nodes, use the following command to:
 
     **Tip:** It may take a few minutes for your node to come back online after reboot.
 
```

Comment:
This was a mistake. Should be covered in this PR https://github.com/dcos/dcos-docs/pull/707/files